mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -399,7 +400,10 @@ export class ExperimentView extends Component {
           </form>
           <div className="ExperimentView-run-buttons">
             <span className="run-count">
-              {runInfos.length} matching {runInfos.length === 1 ? 'run' : 'runs'}
+              {runInfos.length > SEARCH_MAX_RESULTS ?
+                'Showing the latest 1000 matching runs' :
```

Comment:
Can we use `SEARCH_MAX_RESULTS` instead of `1000` in case a user wants to deploy their MLflow UI with a larger limit?

Reply:
Good idea. I will change it.